snuba/api.py

```diff
@@ -18,6 +18,7 @@
 from snuba.split import split_query
 from snuba.datasets.factory import InvalidDatasetError, get_dataset, get_enabled_dataset_names
 from snuba.datasets.schema import local_dataset_mode
+from snuba.schemas import Request, RequestSchema
```

Comment:
`snuba.schemas` might be the wrong place for these classes already.

Reply:
Need to put some more thought into this — I think we need to further decouple Snuba queries/requests from queries executed against the storage layer, and I worry this will muddy the waters a bit more right now.